core/lib/types/src/l2_to_l1_log.rs

```diff
@@ -18,7 +19,7 @@ impl L2ToL1Log {
     /// Legacy upper bound of L2-to-L1 logs per single L1 batch. This is not used as a limit now,
     /// but still determines the minimum number of items in the Merkle tree built from L2-to-L1 logs
     /// for a certain batch.
-    pub const LEGACY_LIMIT_PER_L1_BATCH: usize = 512;
+    pub const LEGACY_LIMIT_PER_L1_BATCH: usize = 2048;
```

Comment:
Should it be changed? If yes then I'm not sure that `LEGACY_LIMIT_PER_L1_BATCH` is a good name for the constant, because we didn't have 2048 as a limit ever, right?